waspc/src/Wasp/AI/GenerateNewProject/WaspFile.hs

```diff
@@ -165,12 +142,3 @@ getWaspFileCompileErrors waspSource =
     <&> either (map showCompileError) (const [])
   where
     showCompileError (errMsg, Ctx {ctxSourceRegion = loc}) = show loc <> ": " <> errMsg
-
-data WaspFile = WaspFile
```

Comment:
Made a generic `FileContent` data type and it's shared between this fixing process and the Prisma fixing process.